ironfish-cli/src/commands/chain/broadcast.ts

```diff
@@ -0,0 +1,41 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+import { CliUx } from '@oclif/core'
+import { IronfishCommand } from '../../command'
+import { RemoteFlags } from '../../flags'
+
+export class BroadcastCommand extends IronfishCommand {
+  static description = `Broadcast a transaction to the network`
+
+  static flags = {
+    ...RemoteFlags,
+  }
+
+  static args = [
+    {
+      name: 'transaction',
+      required: true,
+      parse: (input: string): Promise<string> => Promise.resolve(input.trim()),
```

Comment:
I don't think we need this as it's already done by default in oclif. Will you check and delete this?